src/id.rs

```diff
@@ -257,7 +261,7 @@ impl P2pNode {
 
 impl Debug for P2pNode {
     fn fmt(&self, formatter: &mut Formatter) -> fmt::Result {
-        write!(formatter, "PublicId(name: {})", self.public_id.name())
+        write!(formatter, "{:?}", self.public_id)
```

Comment:
This impl makes the debug output of `P2pNode` indistinguishable from the one of `PublicId`. So I would at least do something like:
```rust
write!(formatter, "P2pNode(name: {})", self.public_id.name())
```
but maybe better to also include the conn info:
```rust
write!(formatter, "P2pNode({} at {})", self.public_id.name(), self.conn_info.peer_addr)
```
or something like that.

Reply:
Yep good point, addressed in latest update